sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-files/e-cert-response-record.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from "@nestjs/common";
+import { ECertResponseRecordIdentification } from "../e-cert-response-record-identification";
+
+/**
+ * Parsing E-Certs feedback records to get the document number and associated
+ * error codes.
+ * The documentation about it is available on the document
+ * 'CSLP-AppendixF2AsReviewed2016-FileLayouts BC Files V3(HAJ-CB EDITS) In ESDC Folder'.
+ */
+@Injectable()
+export abstract class ECertResponseRecord extends ECertResponseRecordIdentification {
+  constructor(line: string, lineNumber: number) {
+    super(line, lineNumber);
+  }
+
+  /**
+   * Financial document number associated with this disbursement.
+   */
+  public get documentNumber(): number {
+    throw new Error(`Method not implemented.`);
+  }
+
+  /**
+   * error code-1 associated with the document number.
```

Comment:
Start the sentences with upper case, please. Same for other places in this file.

Reply:
Please search for `error code-` in the source code. There are still some occurrences.